Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml

```diff
@@ -266,14 +266,14 @@
                             </div>
                             <div class="govuk-date-input__item">
                                 <div class="govuk-form-group">
-                                    <label for="amalgamationdate-month" class="govuk-label govuk-date-input__label">Month</label>
-                                    <input class="govuk-input date-text-month" type="text" v-model="mergeDateMonth" id="amalgamationdate-month">
+                                    <label for="amalgamation-date-month" class="govuk-label govuk-date-input__label">Month</label>
+                                    <input class="govuk-input date-text-month" type="text" v-model="mergeDateMonth" id="amalgamation-date-month">
                                 </div>
                             </div>
                             <div class="govuk-date-input__item">
                             <div class="govuk-form-group">
-                                <label for="amalgamationdate-year" class="govuk-label govuk-date-input__label">Year</label>
-                                    <input class="govuk-input gias-year-input__item date-text-year" type="text" v-model="mergeDateYear" id="amalgamationdate-year">
+                                <label for="amalgamation-date-year" class="govuk-label govuk-date-input__label">Year</label>
```

Comment:
@JaswiniP should the for value be changed here? FYI @RogerHowellDfE

Reply:
My best understanding - this value needs to match the model class

Reply:
See comment on the other one :)